specs/proposals/blob-signing.md

```diff
@@ -0,0 +1,282 @@
+            "size": 845,
+            "annotations": {
+                "org.opencontainers.image.title": "sbom.json"
+            }
+            }
+        ],
+        "annotations": {
+            "org.opencontainers.image.created": "2025-02-18T06:00:00Z"
+        }
+    }
+    ```
+
+    Below is an example of the manifest for the blob signature in the registry:
+
+    ```json
+    {
+        "schemaVersion": 2,
+        "mediaType": "application/vnd.oci.image.manifest.v1+json",
+        "artifactType": "application/vnd.cncf.notary.signature",
+        "config": {
+            "mediaType": "application/vnd.oci.empty.v1+json",
+            "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+            "size": 2,
+            "data": "e30="
+        },
+        "layers": [
+            {
+            "mediaType": "application/cose",
+            "digest": "sha256:6dc2a336b124d625f87deb3f539fa1465414ddad300a898adf4682e530d6f144",
+            "size": 1479,
+            "annotations": {
+                "org.opencontainers.image.title": "sbom.json.cose.sig"
+            }
+            }
+        ],
+        "subject": {
+            "mediaType": "application/vnd.oci.image.manifest.v1+json",
+            "digest": "sha256:57eed31a09556933bddd4e164e1e085a220cf00489477eb98d89feac902eb42b",
+            "size": 553
+        },
+        "annotations": {
+            "io.cncf.notary.blob.signature": "true",
+            "io.cncf.notary.x509chain.thumbprint#S256": "[\"aaaa\", \"bbbb\"]",
+            "org.opencontainers.image.created": "2025-02-18T06:08:45Z"
+        }
+    }
+    ```
+
+    Alternatively, you can also use ORAS tool to push the blob file first, and then use Notation CLI command to push only the signature:
+
+    ```shell
+    oras push myregistry/mypath/mysboms:v1 sbom.json:application/octet-stream 
+    notation blob push --signature sbom.json.cose.sig --reference myregistry/mypath/mysboms:v1
+    ```
+
+    You can also create an oci image layout and then use ORAS tool to copy it to the target registry. 
+
+    ```shell
+    notation blob push --artifact-type "application/spdx+json" sbom.json:"application/json" --signature sbom.json.cose.sig --oci-layout sbom.json:v1
+    oras copy -r --from-oci-layout sbom.json:v1 myregistry/mypath/mysboms:v1
+    ```
+
+    > NOTE:
+    >
+    > Support for OCI image layout in Notation is currently an experimental feature.
+
+3. Verify the signature from the registry:
+
+    - Add the root CA certificate and set up the trust store:
+
+        ```shell
+        notation cert add --type ca --store myCACerts root.crt
+        ```
+
+        Confirm that the cert is added.
+
+        ```shell
+        notation cert ls
+        ```
+
+    - Set up the trust policy:
+
+        ```shell
+        notation blob policy init --name "myBlobPolicy" --trust-store "ca:myCACerts" --trust-identity "x509.subject:C=US,ST=WA,O=wabbit-network.io"
```

Comment:
Again, why do we need to have `policy` command and `blob policy` subcommand? This is suboptimal experience IMHO

Reply:
See this comment https://github.com/notaryproject/notation/pull/1180#discussion_r1962855581

Reply:
Toddy and I discussed offline and agreed that the current proposal for managing trust policies for OCI artifacts and blobs is not optimal.  We should explore a better solution due to potential user confusion with the two policy commands and configuration files. /cc @shizhMSFT @Two-Hearts @JeyJeyGao

We also need to consider the release version, if we starts with v2, we may break the experience later on and require a major version stepping up. One option is that we could do v1.4.0 release first, and improve the experience and other command experience in v2 including trust policy format.